SDK-1009: Change dashboard to hub (#57)
getyoti/yoti-ruby-sdk: README.md

* Fraud list - Verify against  US Social Security Administration Fraud (SSN Fraud) list
* Watch list - Verify against watch lists from the Office of Foreign Assets Control

> To use this functionality you must ensure your application is assigned to your organisation in the Yoti Hub - please see [here](https://www.yoti.com/developers/documentation/#1-creating-an-organisation) for further information.

Reviewer: Same with this one:
```suggestion
To use this functionality you must ensure your application is assigned to your organisation in the Yoti Hub - please see [here](https://developers.yoti.com/yoti-app-integration/yoti-app-integration#step-1-creating-an-organisation) for further information.
```

Author: I've left this URL as is as we are going to update these as a separate task

For the AML check you will need to provide the following:

> * Data provided by Yoti (please ensure you have selected the Given name(s) and Family name attributes from the Data tab in the Yoti Hub)

Reviewer: ```suggestion
* Data provided by Yoti (please ensure you have selected the Given name(s) and Family name attributes for your scenario on the Yoti Hub)
```
